src/BKTClient.ts

```diff
@@ -33,47 +77,67 @@ export class BKTClientImpl implements BKTClient {
   }
 
   stringVariation(featureId: string, defaultValue: string): string {
-    const value = this.getVariationValue(featureId)
```

Comment:
Plz remvoe `getVariationValue()`. It is not referenced by any codes.

Reply:
Thank you for reviewing. 
I removed it https://github.com/bucketeer-io/javascript-client-sdk/pull/155/commits/88dd9ef287c8e26c4c0d0b73120a9c2710686815